docs/spark/tutorials/get-started.md

```diff
@@ -16,104 +16,185 @@ In this tutorial, you learn how to:
 
-```bash
-git clone https://github.com/dotnet/spark.git c:\github\dotnet-spark
-```
+Once you've installed everything and set your environment variables, open a **new** command prompt and run the following command:
+
+`%SPARK_HOME%\bin\spark-submit --version`
+
+If the command runs, printing out version information, you're good to go.
+
+If you receive a `'spark-submit' is not recognized as an internal or external command` error, make sure you opened a **new** command prompt.
+
+### 5. Install .NET for Apache Spark
+
+  * Download the [Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases) release from the .NET for Apache Spark GitHub. For example if you're on a Windows machine and planning to use .NET Core, [download the Windows x64 netcoreapp2.1 release](https://github.com/dotnet/spark/releases/download/v0.5.0/Microsoft.Spark.Worker.netcoreapp2.1.win-x64-0.6.0.zip).
+  * Locate the **Microsoft.Spark.Worker.netcoreapp2.1.win-x64-0.6.0.zip** file that you downloaded.
+  * Right click and select **7-Zip -> Extract files...**
+  * Enter **C:\bin** in the **Extract to** field.
+  * Uncheck the checkbox below the **Extract to** field.
+  * Click **OK**.
+  
+![Install .NET Spark](https://dotnet.microsoft.com/static/images/dotnet-for-spark-extract-with-7-zip.png?v=jwCyum9mL0mGIi4V5zC7yuvLfcj1_nL-QFFD8TClhZk)
+
+### 6. Install WinUtils
+
+.NET for Apache Spark requires WinUtils to be installed alongside Apache Spark.
+
+  * [Download winutils.exe](https://github.com/steveloughran/winutils/blob/master/hadoop-2.7.1/bin/winutils.exe)
+  * Once **winutils.exe** downloads, copy it into **C:\bin\spark-2.4.1-bin-hadoop2.7\bin**.
+
+**Note:** If you are using a different version of Hadoop (which is annotated at the end of your Spark install folder name), you'll need to [select a version of WinUitls](https://github.com/steveloughran/winutils) compatible with your version of Hadoop. 
+
+### 7. Set DOTNET_WORKER_DIR and check dependencies
+
+Run the following command to set the `DOTNET_WORKER_DIR` Environment Variable. This is used by .NET apps to locate .NET for Apache Spark:
+
+`setx DOTNET_WORKER_DIR "C:\bin\Microsoft.Spark.Worker-0.6.0"`
+
+Finally, double-check that you can run `dotnet`, `java`, `mvn`, `spark-shell` from your command line before you move to the next section.
 
 ## Write a .NET for Apache Spark app
 
-1. Open **Visual Studio** and navigate to **File > Create New Project > Console App (.NET Core)**. Name the application **HelloSpark**.
+### 1. Create a console app
 
-2. Install the [Microsoft.Spark NuGet package](https://www.nuget.org/profiles/spark). For more information on installing NuGet packages, see [Different ways to install a NuGet Package](https://docs.microsoft.com/nuget/consume-packages/ways-to-install-a-package).
+In your command prompt, run the following commands to create a new console application:
 
-3. In **Solution Explorer**, open **Program.cs** and write the following C# code:
+```
```

Comment:
An identifier should be added here, either dotnetcli or console.

Reply:
Added console identifier.